### PR TITLE
Fix collapsed accordion bottom margin

### PR DESCRIPTION
**Pull request by ScopeyNZ**
Couldn't really figure out a nice way to do this without moving it to JS. 😞 

Fixes https://github.com/chec/dashboard/issues/274

### CODE REVIEW

**Comment by robbieaverill** [comment]
Fixes https://github.com/chec/dashboard/issues/274?

**Comment by ScopeyNZ** [comment]
Yep, was just going looking for that. Thanks.

**Review by kvisca — APPROVED** [review]
LGTM

File: src/components/ChecAccordion.vue [hunk 1]
```diff
@@ -136,25 +161,22 @@ export default {
...
+    transition: max-height 300ms cubic-bezier(0, 1, 0, 1),
+      margin-bottom 300ms cubic-bezier(0, 1, 0, 1);
   }
 
   &__body {
-    @apply p-4 pb-0;
+    @apply p-4;
 
-    transition: padding-bottom 100ms linear 200ms;
+    //transition: padding-bottom 100ms linear 200ms;
```
Comment (robbieaverill) [inline src/components/ChecAccordion.vue:171]:
```suggestion
```

And previous line (surprised this isn't a linting error)

File: src/components/ChecAccordion.vue [hunk 2]
```diff
@@ -96,6 +106,16 @@ export default {
...
   },
+  mounted() {
+    this.contentHeight = this.$refs.body.clientHeight;
+
+    const vm = this;
+    const resizeObserver = new ResizeObserver(() => {
+      this.contentHeight = vm.$refs.body.clientHeight;
+    });
+
+    resizeObserver.observe(this.$refs.body);
```
Comment (robbieaverill) [inline src/components/ChecAccordion.vue:117]: Do you need to unmount this stuff?

File: src/components/ChecAccordion.vue (hunk 1 above)
Comment (ScopeyNZ) [reply src/components/ChecAccordion.vue:171]: I guess it doesn't know that the content of the comment is useless.

File: src/components/ChecAccordion.vue (hunk 2 above)
Comment (ScopeyNZ) [reply src/components/ChecAccordion.vue]: Wouldn't hurt

**Pull request merged**